example/SimpleApp/platformio.ini

```diff
@@ -22,11 +22,13 @@ build_flags =
     -D VERSION=\"1.0.0\"
     -D PIO_ENV=\"${PIOENV}\"
     -D LOG_LOCAL_LEVEL=ESP_LOG_DEBUG
-;extra_scripts =
-;    pre:../../script/merge_factory.py
+extra_scripts =
+    pre:../../script/add_factory_to_extra_images.py
 monitor_speed = 115200
 monitor_filters = esp32_exception_decoder
 upload_protocol = esptool
+custom_factory_name = factory ;must be name of the partition and the binary file

```

Comment:
Why its used for the name of the partition and the name of the binary? Any rational behind?